### ISSUE TITLE
weather@mockturtl: update frequency should perhaps be lowered

### ISSUE THREAD

**Issue by Pjotr123** [linuxmint/cinnamon-spices-applets#6092]
### Applet name and maintainer

weather@mockturtl @Gr3q

### What would you like to see?

On the Linux Mint forum, there have been several complaints about the high update frequency of Weather (weather@mockturtl). Because the restart of Cinnamon that such an update causes, is disruptive in some setups.

It might therefore be a good idea to bundle language updates in order to reduce the number of Weather updates. Perhaps a maximum of one update a week would be a good idea?

**Comment by rcalixte** [comment]
Development and translations are handled individually by translators and the main developer. Some changes also require previous changes as well. Once code changes are merged, they are deployed from GitHub to the Spices website and made available for end users.

For those who are curious, the development can be tracked [here](https://github.com/linuxmint/cinnamon-spices-applets/commits/master/weather%40mockturtl). As you can see, there are times where things are busier and times where things are calmer.



> Perhaps a maximum of one update a week would be a good idea?

This would mean that updates would be distributed and that translations would be held. I struggle with the idea, considering the potential negative impact on end users who need those translations. Maybe there is a better idea or a better workaround. For example, are the restarts after updates a requirement?

**Comment by Pjotr123** [comment]
Unfortunately, the Cinnamon restart after an applet update appears to be a requirement.... It always happens on my Linux Mint 21.3 Cinnamon machines after Weather got updated (Weather is the only applet I ever install).

I'm a translator myself (for Dutch), and personally I like the quick updates that Weather gets, because I never have to wait long before my translations are applied. And on my simple straightforward setups a Cinnamon restart is never disruptive.

But apparently a Cinnamon restart can be disruptive on more complex setups. Which is of course unfortunate, especially if the frequency is high....

**Comment by binyominzeev** [comment]
Yes please this would be great. Is it possible to somehow gather together all updates and issue them only once every week (like a digest email)?

https://forums.linuxmint.com/viewtopic.php?t=413034

Thank you very much!
